src/components/pageComponents/PageToolbar/PageToolbar.tsx

```diff
@@ -129,24 +129,22 @@ export const PageToolbar = () => {
             <MenuItem sx={styles.menuItem}>{link.text}</MenuItem>
           </Link>
         ))}
-        {isHamalUser && (
-          <>
-            <Divider variant="middle" sx={styles.divider} />
-            {hamalLinks.map((link) => (
-              <Link
-                key={link.text}
-                to={link.href}
-                style={linkStyles}
-                onClick={handleCloseMenu}
-              >
-                <MenuItem sx={styles.menuItem}>
-                  {link.text}
-                  <Chip label={'חמ"ל'} color="primary" sx={styles.chip} />
-                </MenuItem>
-              </Link>
-            ))}
-          </>
-        )}
+        {/* can't use Fragments inside Menu component */}
+        {isHamalUser && <Divider variant="middle" sx={styles.divider} />}
```

Comment:
u can wrap it in one conditional rendering just wrap them both with <></>

Reply:
This is what I originally did. but It keeps giving me warnings that I can't use <></> inside a Menu component. so I changed it to make the warnings disapear